internal/secrets/gcpkms/kms_test.go

```diff
@@ -43,7 +43,7 @@ type harness struct {
 }
 
 func (h *harness) MakeDriver(ctx context.Context) (driver.Crypter, error) {
```

Comment:
The driver tests should be updated to use the concrete type instead of the driver directly; right now your test coverage of the concrete type is 0%.